types/luaparse/lib/ast.d.ts

```diff
@@ -136,7 +136,7 @@ export interface TableConstructorExpression extends Base<"TableConstructorExpres
 }
 
 export interface UnaryExpression extends Base<"UnaryExpression"> {
-    operator: "#" | "not";
+    operator: "#" | "not" | '-';
```

Comment:
Can this string use double quotes for consistency with the rest of the file? Also I believe this change is already in another PR so you may want to remove it to avoid conflicts.